src/client/components/Inputs/Select/Select.tsx

```diff
@@ -16,6 +16,9 @@ const Select: React.FC<SelectProps> = (props) => {
   const value = useValue(props)
   const onChange = useOnChange(props)
 
+  // Keep menu open when selecting an option in multi-select mode
+  const closeMenuOnSelect = !isMulti
```

Comment:
It looks weird for single select where you expect it to close on select
On multi select it makes sense - the user might want to select more than one option, especially when selecting many options it might be rather annoying always to reopen

https://github.com/openforis/fra-platform/assets/5508251/523172af-03a6-4bf6-8a15-b8a50ea33b26